sdk/contentsafety/azure-ai-contentsafety/tests/conftest.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from devtools_testutils import test_proxy, add_general_string_sanitizer
+import os
+
+# autouse=True will trigger this fixture on each pytest run, even if it's not explicitly used by a test method
+@pytest.fixture(scope="session", autouse=True)
+def start_proxy(test_proxy):
+    return
+
+
+@pytest.fixture(scope="session", autouse=True)
+def add_sanitizers(test_proxy):
+    content_safety_endpoint = os.environ.get("CONTENT_SAFETY_KEY", "00000000000000000000000000000000")
+    content_safety_key = os.environ.get(
+        "CONTENT_SAFETY_ENDPOINT", "https://fake_cs_resource.cognitiveservices.azure.com/"
+    )
```

Comment:
Same here, I think this is all affecting CI:
```suggestion
    content_safety_endpoint = os.environ.get(
        "CONTENT_SAFETY_ENDPOINT", "https://fake_cs_resource.cognitiveservices.azure.com"
    )
```